examples/chip-tool/commands/common/Commands.cpp

```diff
@@ -47,8 +47,9 @@ int Commands::Run(NodeId localId, NodeId remoteId, int argc, char ** argv)
     {
 
         ChipDeviceController dc;
+        ChipToolPersistentStorageDelegate storage;
 
-        err = dc.Init(localId);
```

Comment:
If the one-arg version of `Init` doesn't actually work in practice, can we remove it?